entrypoint.rb

```diff
@@ -10,12 +10,13 @@
 verbose = ENV["VERBOSE"]
 wait = ENV["WAIT_INTERVAL"]
 workflow_name = ENV["RUNNING_WORKFLOW_NAME"]
+api_endpoint = ENV['API_ENDPOINT'] || 'https://api.github.com'
 
 GithubChecksVerifier.configure do |config|
   config.allowed_conclusions = allowed_conclusions.split(",").map(&:strip)
   config.check_name = check_name
   config.check_regexp = check_regexp
-  config.client = Octokit::Client.new(access_token: token)
+  config.client = Octokit::Client.new(access_token: token, api_endpoint: api_endpoint)
```

Comment:
An alternative to avoid instantiating a local variable:

```suggestion
  if ENV.fetch("API_ENDPOINT", "").strip.empty?
    config.client = Octokit::Client.new(access_token: token)
  else
    config.client = Octokit::Client.new(access_token: token, api_endpoint: ENV["API_ENDPOINT"])
  end
```

If you go with that solution, you can ignore my previous comment ☝️  (and get rid of the `api_endpoint` variable definition)

Reply:
went with the first fix since it was more inline with the pattern